### PR TITLE
chore: updated the .env setup instructions

### PR DESCRIPTION
**Pull request by cgen01**
#### Brief description

Added some ENV setup instructions

#### Updates

Updated the README.md file

---

#### Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] My change requires a change to the documentation.
-   [x] I have updated the documentation accordingly.
-   [ ] I have added tests to cover my changes.

---

#### Contributing policy agreement

-   [x] I have read the [**CONTRIBUTING**](https://github.com/troup-io/troup-server/blob/master/CONTRIBUTING.md) document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,23 @@ The Troup client is the frontend app that helps the user address problems that w
...
+NODE_PATH="./src"
+DATABASE_URL="postgresql://prisma:prisma@localhost:5432/postgres?schema=public"
+```
+
+The second `.env` file should be created in the `prisma` folder and contains the following information:
+
+```
+DATABASE_URL="postgresql://prisma:prisma@localhost:5432/postgres?schema=public"
+```
+
```
Comment (samrith-s) [inline README.md]: I have added two `example.env` files in the repo. These can be used to create the actual `.env` files. We should update the content to reflect the same, and redirect to those files.

```suggestion
## Setting up environment variables

You will need to create two `.env` files in the root and the `prisma` folder. The instructions on setting those up are available within each example:
- [Root environment variables](https://github.com/troup-io/troup-server/blob/master/example.env)
- [Prisma environment variables](https://github.com/troup-io/troup-server/blob/master/prisma/example.env)
```

**Review by samrith-s — CHANGES_REQUESTED** [review]
@cgen01 This is great, thanks! Just a small change which I think will better reflect how we manage environment variables and provide more context. 😄

File: README.md (hunk 1 above)
Comment (cgen01) [reply README.md]: Perfect, that makes sense! Thanks

**Review by samrith-s — APPROVED** [review]
This is great! Thank you so much! 😄

**Pull request merged**